src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CaseRepository.java

```diff
@@ -15,6 +15,8 @@
 
     Optional<Claim> getClaimByExternalId(String externalId, String authorisation);
 
+    Optional<Claim> getClosedCaseByExternalId(String externalId, String authorisation);
```

Comment:
Would prefer to have something like `Optional<Claim> getCase(String externalId, String authorisation, CaseState caseState)` instead of `getClosedCaseByExternalId(..)`